docs/config/policies.md

```diff
@@ -161,15 +162,18 @@ during the cleanup process.
         name: "mypolicy"
         docker:
           keepSemver: true
+          repoRetentionCount: 3
           exclude:
             - "latest"
     ```
 
 !!! abstract "Environment variables"
     * `ATFCLNP_POLICIES_<KEY>_DOCKER_KEEPSEMVER`
     * `ATFCLNP_POLICIES_<KEY>_DOCKER_EXCLUDE`
+    * `ATFCLNP_POLICIES_<KEY>_DOCKER_REPORETENTIONCOUNT`
 
-| Name               | Default       | Description   |
-|--------------------|---------------|---------------|
-| `keepSemver`       |               | Do not remove tags matching a semver compliant pattern |
-| `exclude`          |               | List of tags matching an expression to exclude |
+| Name                 | Default       | Description   |
+|----------------------|---------------|---------------|
+| `keepSemver`         |               | Do not remove tags matching a semver compliant pattern |
+| `repoRetentionCount` |               | The number of images that should be kept in each docker repository regardless of `lastModified` and `lastDownloaded` |
```

Comment:
It should be a global policy option in: https://github.com/crazy-max/artifactory-cleanup/blob/cdd2cea6e2203052a72d70a68ee0e59be657c84a/internal/config/policies.go#L12-L22

Not just tied to the docker policy.